src/main.cpp

```diff
@@ -109,6 +110,11 @@ QCoreApplication* createApplication(int& argc, char* argv[], unsigned int& actio
             if (tolower(argument) == 'q') {
                 action |= 4;
             }
+        } else {
```

Comment:
We might want to review this to specify a command line argument as an introducer for a profile to load - with another one to NOT autoload any that are already set to autorun on start-up - it will make the string to fire up the Mudlet instance a bit longer though...

Reply:
i.e. instead of something like `/path/to/executable/mudlet game.domain.org:portNumber` we use `/path/to/executable/mudlet -l game.domain.org:portNumber` so that the `-l` (or `--load`) option indicates that what follows is a Game server address to connect to - this makes it a bit easier to parse the command line as I understand it. It is more flexible if the user wants to open more than one profile at a time as then, if there are other options that can (in the future) be applied from the command line, the options can be applied to a specific profile.

Reply:
Agree!